packages/server/postgres/queries/src/getAtlassianAuthsByUserId.sql

```diff
@@ -0,0 +1,5 @@
+/*
+  @name getAtlassianAuthsByUserIdQuery
+*/
+SELECT * from "AtlassianAuth"
+WHERE "userId" = :userId AND "isActive" = TRUE;
```

Comment:
> Do we want to add an index to userId column in the AtlassianAuth table?

Since we already have a composite index on `("userId", "teamId")`, this shouldn't be necessary. The way that postgres indexes are implemented under the hood, querying for `userId` will be able to use the composite index efficiently. If we were to query using only `teamId`, that's a different story. more info: https://stackoverflow.com/questions/11352056/postgresql-composite-primary-key

Reply:
Thank you for the explanation! We came to the same conclusion with Matt on our 1:1 and it's really good to know it works that way. 👍